internal/github/github.go

```diff
@@ -0,0 +1,128 @@
+package github
+
+import (
+	"context"
+	"encoding/base64"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/bradleyfalzon/ghinstallation/v2"
+	"github.com/google/go-github/v64/github"
+	"golang.org/x/crypto/nacl/box"
+)
+
+// Client defines the interface for interacting with GitHub
```

Comment:
What in the copilot is this, does not add anything. It is in the GitHub file and already a client interface